### PR TITLE
fix: add labels to examples

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR summary
This PR adds labels to example output, so it is easier to identify which output section belongs to which method.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by pyrooka — APPROVED** [review]
For me, it looks good.

File: examples/test_case_management_v1_examples.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def test_create_case_example(self):
                 severity=4,
             ).get_result()
 
-            print(json.dumps(case, indent=2))
+            print('\ncreate_case() result:\n', json.dumps(case, indent=2))
```
Comment (pyrooka) [inline examples/test_case_management_v1_examples.py:112]: Concatenating strings in Python's `print` function with comma (,) adds a space between them. This brokes the printed result a little bit:
```
upload_file() result:
 {
  "id": "xxx",
  "filename": "example.log",
  "size_in_bytes": 42,
  "created_at": "2021-03-23 16:52:19",
  "url": "xxx"
}
```
```suggestion
            print('\ncreate_case() result:\n' + json.dumps(case, indent=2))
```

**Review by pyrooka — APPROVED** [review]
I approve it, but let's discuss my suggestion.

File: examples/test_case_management_v1_examples.py (hunk 1 above)
Comment (padamstx) [reply examples/test_case_management_v1_examples.py:112]: I like @pyrooka 's suggestion :)

**Review by padamstx — APPROVED** [review]
LGTM

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**